homeshick.fish

```diff
@@ -7,6 +7,7 @@ function homeshick
 	if test \( (count $argv) = 2 -a $argv[1] = "cd" \)
 		cd "$HOME/.homesick/repos/$argv[2]"
 	else
+		set -x HOMESHICK_DIR $HOME/.homesick/repos/homeshick
```

Comment:
Yes, it will. But only if it's already defined when the `homeshick` function is invoked. Strange...
Anyway, I have fixed it now.